libs/@local/harpc/client/typescript/src/types/ResponseKind.ts

```diff
@@ -0,0 +1,156 @@
+import type { FastCheck } from "effect";
+import {
+  Effect,
+  Equal,
+  Hash,
+  Inspectable,
+  pipe,
+  Pipeable,
+  Predicate,
+} from "effect";
+
+import { createProto, encodeDual } from "../utils.js";
+import * as Buffer from "../wire-protocol/Buffer.js";
+import * as ErrorCode from "./ErrorCode.js";
+
+const TypeId: unique symbol = Symbol(
+  "@local/harpc-client/wire-protocol/types/ResponseKind",
+);
+
+export type TypeId = typeof TypeId;
+
+export interface Ok
+  extends Equal.Equal,
+    Inspectable.Inspectable,
+    Pipeable.Pipeable {
+  readonly [TypeId]: TypeId;
+  readonly _tag: "Ok";
+}
+
+const OkProto: Ok = {
+  [TypeId]: TypeId,
+  _tag: "Ok",
+
+  [Equal.symbol](this: Ok, that: Equal.Equal) {
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
+    return isOk(that);
+  },
+
+  [Hash.symbol](this: Ok) {
+    return pipe(
+      Hash.hash(this[TypeId]), //
```

Comment:
I put the empty comment there to force a line break, to make this more readable, but can also remove it if wanted.

Reply:
I think it's fine in this one instance, but generally if we want to have `pipe` always format across lines we'd be better off having a Prettier plugin which does it, because scattering empty comments across the codebase will look a bit weird.